iib/web/models.py

```diff
@@ -789,7 +790,7 @@ def _from_json(
 
         # Validate binary_image is correctly provided
         binary_image = request_kwargs.pop('binary_image', None)
-        if binary_image is not None and not isinstance(binary_image, str):
+        if binary_image and not isinstance(binary_image, str):
```

Comment:
This will not throw an error if binary_image is passed as `False`. Could you please change it back to what it was?